simulation/executor/util.go

```diff
@@ -28,7 +37,7 @@ func getTestingMode(tb testing.TB) (testingMode bool, t *testing.T, b *testing.B
 //  - "over stuffed" blocks with average size of 2 * avgblocksize,
 //  - normal sized blocks, hitting avgBlocksize on average,
 //  - and empty blocks, with no txs / only txs scheduled from the past.
-func getBlockSize(simCtx *simtypes.SimCtx, params Params, lastBlockSizeState, avgBlockSize int) (state, blockSize int) {
+func getBlockSize(simCtx *osmosim.SimCtx, params Params, lastBlockSizeState, avgBlockSize int) (state, blockSize int) {
```

Comment:
Lets keep this as simtypes and move the new code to its own file

Reply:
I think a prematurely opened this PR as I made quite a few more changes in which I reference SimCtx via osmosim instead of through sdk simulator. If it is okay with you I will make one more large commit, and from there could you let me know if this point still stands?